libvirt/domain.go

```diff
@@ -18,12 +18,6 @@ import (
 	"github.com/libvirt/libvirt-go-xml"
 )
 
-// deprecated, now defaults to not use it, but we warn the user
-const skipQemuAgentEnvVar = "TF_SKIP_QEMU_AGENT"
```

Comment:
can you remove also https://github.com/dmacvicar/terraform-provider-libvirt/blob/b11e5b79ff40dc8f03d762d92226fabe315955dc/travis/run-tests-inside-guest#L21